english/marshmallow.md

```diff
@@ -1,3 +1,7 @@
+Once upon a time many authors collaborated to make meaningless linked story.
+
+[start](mad/mad.md) your journey into world of mad or spend your time [wisely](http://www.pythonchallenge.com/)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.